src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -205,6 +176,9 @@ public ServerConfigDocument(CommonLoggerI log, Map<String, File> libertyDirPrope
         props = new Properties();
         if (initProperties != null) props.putAll(initProperties);
         defaultProps = new Properties();
+        if (originalServerXMLFile != null) {
```

Comment:
Why are we safeguarding setting this field to null, when it defaults to null anyway. Seems unnecessary.